sklearn/compose/_column_transformer.py

```diff
@@ -479,16 +503,17 @@ def _check_key_type(key, superclass):
     return False
 
 
-def _hstack(X):
+def _hstack(X, sparse_):
```

Comment:
Nitpick: Is it worth implementing sparse_threshold here (and then setting `sparse_output_ = sp.issparse(Xt)`)?

Might make it a little easier to test, particularly the boundary case.

Reply:
I don't fully follow how that would make it easier to test?